charts/simple-app/templates/ingress/ingress.yaml

```diff
@@ -12,16 +12,6 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  {{- if .Values.ingress.tls }}
```

Comment:
Removed this boilerplate because we don't use this anywhere - its specific to the `nginx` ingress controller.